NEWS.md

```diff
@@ -1,3 +1,7 @@
+# dust 0.8.7
+
+* Invalid inputs in `rbinom` are converted into exceptions which are safely thrown even from parallel code. After an error, the object will be in an inconsistent state as not all particles will have completed their run and you must reset the object (via `reset()` or `set_state()`) to continue (#190)
```

Comment:
The second sentence should probably be in docs somewhere, rather than/as well as news

Reply:
good shout, updated the docs (though this also makes it clear they need an overhaul - see #207)